research-software-directory/RSD-as-a-service
# Fix remote markdown layout

# Pull request Title

Closes #691

Changes proposed in this pull request:
* When using remote markdown in RSD the layout deviates from the one shown on Github/Gitlab. There are too many white spaces because we enabled remarkBreaks plugin. For remote markdown this is not correct.
* The badges are shown one per line due to image block display.

How to test:
* `make start` to rebuild all
* create new software or open existing one.
* use remove markdown for RSD: https://raw.githubusercontent.com/research-software-directory/RSD-as-a-service/main/README.md
* confirm the layout is [similar to one on Github](https://github.com/research-software-directory/RSD-as-a-service/blob/main/README.md)

## Incorrect layout 
see image in issue #691 

## Improved layout for remove markdown
![image](https://user-images.githubusercontent.com/9204081/205516776-5fc7ab1f-74c1-48e8-8fcd-22df3393f35d.png)

PR Checklist:
